packages/mui-base/src/Button/Button.spec.tsx

```diff
@@ -24,24 +24,31 @@ const polymorphicComponentTest = () => {
       return <div />;
     };
 
+  const Root = function Root() {
+    return <div />;
+  };
+
   return (
     <div>
       {/* @ts-expect-error */}
       <Button invalidProp={0} />
 
-      <Button component="a" href="#" />
+      <Button slots={{ root: 'a' }} href="#" />
 
-      <Button component={CustomComponent} stringProp="test" numberProp={0} />
```

Comment:
I think we should keep this?

Reply:
Yup, it should stay (with `slots` instead of `component`, that is).